src/main/resources/db/migration/V20231218_47__GA-47_new_access_types_table.sql

```diff
@@ -0,0 +1,14 @@
+-- create table
```

Comment:
comments left over?

Reply:
Comments are there to say that PK/FK relationship is automatically created.